src/routes/boot.ts

```diff
@@ -519,6 +519,7 @@ const loggedInBoot = async ({
         image: mapCloudinaryUrl(user.image),
         cover: mapCloudinaryUrl(user.cover),
         defaultFeedId: isPlus ? user.defaultFeedId : null,
+        shouldShowGiftPlus: Boolean(user?.flags?.showPlusGift),
```

Comment:
I think we should expose it as flags: { showPlusGift } and then when you use update user mutation on frontend it will automatically invalidate the object as well.

Reply:
Updated ✔️